rpc/server_auth.go

```diff
@@ -236,6 +272,17 @@ var validSigningMethods = []string{
 	"RS256",
 }
 
+// tryAuth is called for public methods where auth is not required but preferable.
+func (ss *simpleServer) tryAuth(ctx context.Context) (context.Context, error) {
```

Comment:
@michaellee1019 what do you think of this?

Reply:
much better! It adds a ton of clarity.